tests/vmi_servers.go

```diff
@@ -4,37 +4,37 @@ import (
 	"fmt"
 	"time"
 
-	expect "github.com/google/goexpect"
 	. "github.com/onsi/gomega"
 
 	v1 "kubevirt.io/client-go/api/v1"
-	"kubevirt.io/client-go/log"
 )
 
+type Server string
+
+const (
+	HTTPServer = Server("screen -d -m nc -klp %d -e echo -e \"Hello World!\"\n")
+	TCPServer  = Server("screen -d -m nc -klp %d -e echo -e \"HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!\"\n")
```

Comment:
You've got this wrong. What you're calling HTTPserver is indeed the TCP server and vice-versa.

Reply:
I has just discover it, will fix asap.

Reply:
Done